src/pages/HomePage/FeaturedProducts/FeaturedProducts.css

```diff
@@ -0,0 +1,90 @@
+.FeaturedProducts {
+  max-width: 1177px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 129px;
+}
+.FeaturedProducts h1 {
+  text-align: center;
+  font-size: 42px;
+  line-height: 49px;
+  color: #1a0b5b;
```

Comment:
please create "_variables.scss" file if you don't have it yet, and declare variables for colors there
after that you've got to use those variables in this file